crates/cashu/Cargo.toml

```
@@ -42,6 +42,7 @@ url = { workspace = true }
 itertools = "0.12.0"
 thiserror = { workspace = true }
 uuid = { version = "1.6.1", features = ["v4"] }
+ciborium = "0.2.2"
```

I would recommend always adding a dependency with the default-features set to `false` and than manually importing the features that you want to use in the project. This helps with:

* Better control over dependencies
* Less dependency conflicts
* Reduced binary size 
* Improved compile times

For this specific one:
```suggestion
ciborium = { version = "0.2.2", default-features = false, features = ["std"] }
```